srcpkgs/ballerburg/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ballerburg'
 pkgname=ballerburg
-version=1.2.1
-revision=2
+version=1.2.3
+revision=1
 build_style=cmake
 hostmakedepends="which cmake sdl12-compat-devel"
 makedepends="sdl12-compat-devel"
```

Comment:
```suggestion
hostmakedepends="pkg-config"
makedepends="SDL2-devel"
```

Reply:
If I do that I get
```
[0/1] Re-running CMake...
/bin/sh: line 1: /usr/bin/cmake: No such file or directory
```
Adding back `cmake` to `hostmakedepends` it compiles.

Reply:
I cannot reproduce that behavior locally, cmake templates specifically shouldn't need the `cmake` package, we have `cmake-bootstrap` for that purpose. Probably you need to do `./xbps-src clean`

Reply:
`xbps-src clean` fixed it, so I removed `cmake` and did a push again, thank you.